src/config/configuration.ts

```diff
@@ -46,7 +47,9 @@ const configuration = () => {
 
   // Register built-in job actions
   registerDefaultActions();
-  const job_configs: Promise<JobConfig[]> = loadJobConfig("src/jobs/config/jobConfig.createExample.json");
+  const job_configs: Promise<JobConfig[]> = loadJobConfig(
+    "src/jobs/config/jobConfig.createExample.json",
```

Comment:
We should brainstorm about how to remove the hardcoding and make it more site configurable.

Reply:
Yes, there's a lot of little things like that that will be quick to implement but need to be handled before release.